sjb/generated/merge_pull_request_origin_web_console.xml

```diff
@@ -407,12 +407,6 @@ fi
       <defaultExcludes>true</defaultExcludes>
       <caseSensitive>true</caseSensitive>
     </hudson.tasks.ArtifactArchiver>
-    <hudson.tasks.junit.JUnitResultArchiver plugin="junit@1.13">
```

Comment:
@stevekuznetsov is it OK that this job was impacted?

Reply:
no

Reply:
Fixed!